internal/satellite/module/gatherer/api/config.go

```diff
@@ -33,6 +33,5 @@ type GathererConfig struct {
 	ServerName     string        `mapstructure:"server_name"` // depends on which server
 
 	// FetcherGatherer
-	FetcherConfig plugin.Config `mapstructure:"fetcher"`        // fetcher plugin config
-	FetchInterval int           `mapstructure:"fetch_interval"` // fetch interval, the time unit is millisecond
```

Comment:
FYI @EvanLjp We discussed and decided how to fetch, and what is the period, should be determined by the fetcher itself. A global config and timer don't make sense.

Reply:
Yes， fetchInterval now is set in the implements of fetcher.